Code/Tools/Libs/GuiFoundation/NodeEditor/NodeScene.moc.h

```diff
@@ -35,6 +35,28 @@ class EZ_GUIFOUNDATION_DLL ezQtNodeScene : public QGraphicsScene
   static ezRttiMappedObjectFactory<ezQtConnection>& GetConnectionFactory();
   static ezVec2 GetLastMouseInteractionPos() { return s_LastMouseInteraction; }
 
+  struct VisualStyleFlags
```

Comment:
Currently these are flags for on/off, but if in the future we were to add other modes for the lines or the arrows, this wouldn't be nice to extend. Might be better to have an enum for LineStyle and LineEndStyle or so.